steps/xcode-project-info/2.1.0/step.yml

```diff
@@ -0,0 +1,54 @@
+title: "Xcode Project Info"
+summary: Extracts Xcode project info data to environment variables.
+description: |-
+  This step will simply extract Xcode project information (Version and Build),
+  and export them into environment variables prefixed with **XPI_**.
+
+  Afterwards, you can use these environment variables in other steps (ie. sending message on Slack).
+website: https://github.com/tadija/bitrise-step-xcode-project-info
+source_code_url: https://github.com/tadija/bitrise-step-xcode-project-info
+support_url: https://github.com/tadija/bitrise-step-xcode-project-info/issues
```

Comment:
The step.yml has some of missing fields: 
- published_at
- source:
  - git
  - commit

Please always do a step share with our stepman. Please read the share process here: 
https://devcenter.bitrise.io/contributors/sharing-steps-with-all-bitrise-users/